README.md

```diff
@@ -2,19 +2,41 @@
 
 1.
   * Create a Human class that takes in a name and age.
-  * Add the function `ageOneYear` that ages the human. 
+  * Add the function `ageOneYear` that ages the human.
   * Add the function `eating`, that logs "mmm, mmm, mmm, I'm love'n it".
   * Create an instance of the Human class.
     * console log your humans age
     * call ageOneYear on your human
     * console log their age again.
     * call eating on your human.
-    
-2. 
+
+    class Human{
+      constructor(name, age){
+        this.name = name;
+        this.age = age;
+      }
+      ageOneYear(num){
+        this.age++
+        return this.age
+      }
+      eating(str){
```

Comment:
Just a small nitpick, but the eating function should log "mmm, mmm, mmm, I'm love'n it" instead of returning it. When a question calls for you to log something, 9/10 it means to console.log the result within the function itself.